test/util/testutil.go

```diff
@@ -51,11 +52,11 @@ func waitAndGetOneCertificateRequestInNamespace(ctx context.Context, client cmcl
 	return req, nil
 }
 
-func IssueOneRequest(t *testing.T, client cmclient.Interface, namespace string, stopCh <-chan struct{}, cert, ca []byte) {
-	if err := wait.PollUntil(time.Millisecond*50, func() (done bool, err error) {
-		req, err := waitAndGetOneCertificateRequestInNamespace(context.TODO(), client, namespace)
+func IssueOneRequest(ctx context.Context, t *testing.T, client cmclient.Interface, namespace string, cert, ca []byte) {
+	if err := retry.RetryOnConflict(retry.DefaultRetry, func() error {
```

Comment:
We were using `wait.PollUntil` before, but never returned `done=false` without also returning an error.

Reply:
Wait, I just realised that the goal was to continuously issue certificates instead of retrying.
Fixed now.